drivers/bridge/bridge.go

```diff
@@ -782,7 +782,9 @@ func (d *driver) deleteNetwork(nid string) error {
 			logrus.Warn(err)
 		}
 		if link, err := d.nlh.LinkByName(ep.srcName); err == nil {
-			d.nlh.LinkDel(link)
+			if dellinkerr := d.nlh.LinkDel(link); dellinkerr != nil {
```

Comment:
you can also simply use err := d.nlh.LinkDel(link) err will shadows the other err but is fine

Reply:
ok